components/Navigation.js

```diff
@@ -0,0 +1,233 @@
+    width: "200px",
+    margin: theme.spacing(2),
+  },
+  submenuItem: {
+    color: "#007cc4",
+  },
+  nested: {
+    paddingLeft: theme.spacing(4),
+  },
+}));
+
+export default function Navigation() {
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const mdUp = useMediaQuery(theme.breakpoints.up("md"));
+
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [dropdownOpen, setDropDownOpen] = React.useState(false);
+
+  const open = !!anchorEl;
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const handleClick = (e) => {
+    setAnchorEl(e.currentTarget);
+  };
+
+  const handleDropdownClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleDropdownOpen = () => {
+    setDropDownOpen(!dropdownOpen);
+  };
+
+  const navigationItems = [
+    {
+      text: "Home",
+      url: "/",
+    },
+    {
+      text: "About Us",
+      url: "/about",
+    },
+    {
+      text: "Services",
+      url: "/services",
+      sublinks: [
+        {
+          text: "Consolidated Load",
+          url: "/services-consolidated-load",
+        },
+        {
+          text: "Full Load",
+          url: "/services-full-load",
+        },
+        {
+          text: "Clearing",
+          url: "/services-clearing",
+        },
+      ],
+    },
+    {
+      text: "Quote",
+      url: "/quote",
+    },
+  ];
+
+  const drawer = (
+    <div className={classes.root}>
+      <List component="nav" aria-aria-labelledby="nested-list-subheader" className={classes.root}>
+        {navigationItems.map((navItem) => {
+          if (typeof navItem.sublinks !== "undefined") {
+            return (
+              <List key={navItem.text}>
+                <ListItem button key={navItem.text} onClick={handleDropdownOpen}>
+                  <ListItemText primary={navItem.text} />
+                  {dropdownOpen ? <ExpandLess /> : <ExpandMore />}
+                </ListItem>
+                <Collapse in={dropdownOpen} timeout="auto" entered={classes.nested} unmountOnExit>
+                  <List component="div" disablePadding>
+                    {navItem.sublinks.map((subNavItem) => (
+                      <Link href={subNavItem.url} key={subNavItem.text}>
+                        <ListItem button className={classes.nested}>
+                          <ListItemText primary={subNavItem.text} />
+                        </ListItem>
+                      </Link>
+                    ))}
+                  </List>
+                </Collapse>
+              </List>
+            );
+          } else {
+            return (
+              <ListItem button key={navItem.text}>
+                <Link href={navItem.url}>
+                  <ListItemText primary={navItem.text}></ListItemText>
+                </Link>
+              </ListItem>
+            );
+          }
+        })}
+      </List>
+    </div>
+  );
+
+  const navigationBar = (
```

Comment:
Why is this stored in its own variable? Might as well just add it to the rest of our markup below.